hw/ip/keymgr/doc/theory_of_operation.md

```diff
@@ -92,6 +94,7 @@ This state is reached through another invocation of the KMAC operation using the
 The output of the KMAC operation replaces the previous value of the internal key, and the new value becomes the `OwnerRootKey`.
 
 The relevant inputs are:
+*  `OwnerRootSecret`: Secret seed from flash.
```

Comment:
```suggestion
*   `OwnerSecret`: Secret seed from flash. Rotation of which is enforced by the SiliconCreator through the ROM_EXT.
```

Reply:
@ballifatih can you fix this in a followup? and update the diagram again (https://docs.google.com/drawings/d/1eemXnq7QO7rFsQNoWRacg99cJXr-x8IwwvWbCw9xtao/edit) it has changed just a tad